org.ksnip.ksnip.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: org.ksnip.ksnip
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+command: ksnip
+rename-icon: ksnip
+rename-appdata-file: ksnip.appdata.xml
+rename-desktop-file: ksnip.desktop
```

Comment:
Not a blocker: but this could be renamed upstream as well

Reply:
What would be here the correct name?  `ksnip.ksnip.appdata.xml`?

Reply:
Would be `org.ksnip.ksnip.appdata.xml`?